src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -77,13 +77,17 @@ string GetVisualStudio2017Directory ()
 			get {
 				if (IsUnix) {
 					var outdir = Environment.GetEnvironmentVariable ("XA_BUILD_OUTPUT_PATH");
+					var configuraton = Environment.GetEnvironmentVariable ("CONFIGURATION") ?? "Debug";
```

Comment:
Should this be `?? "Debug"`, or should it be based on `#if DEBUG`, so that a Release build looks in `bin/Release`?

The monodroid build environment won't have `bin/Debug`.